wiring/src/spark_wiring_print.cpp

```diff
@@ -240,3 +242,29 @@ size_t Print::printFloat(double number, uint8_t digits)
 
   return n;
 }
+
+size_t Print::printf_impl(bool newline, const char* format, ...)
+{
+    char test[5];
+    va_list marker;
+    va_start(marker, format);
+    size_t n = vsnprintf(test, 1, format, marker);
```

Comment:
Doesn't passing `1` here mean that test will never contain the correct string for `n<5`?